blank_react_project/package.json

```diff
@@ -45,7 +45,13 @@
     },
     "setupFiles": [
       "<rootDir>/src/jest.init.js"
-    ]
+    ],
+    "testEnvironment": "near-shell/test_environment",
+    "testPathIgnorePatterns": [
+      "<rootDir>/assembly/",
+      "<rootDir>/node_modules/"
+    ],
+    "runInBand": true
```

Comment:
👍

Reply:
looks like they haven't added the ability for this to be in config yet, weird.
https://github.com/facebook/jest/issues/3215